client.go

```diff
@@ -69,3 +79,19 @@ const (
 // Job is a function that will execute like a cron job after a
 // certain amount of time to perform some kind of task.
 type Job func(Bot, Channel) error
+
+//BroadcastFilter returns true if message can be broadcasted to bot and Channel
+type CanBeBroadcasted func(botID string, channel Channel, msg Sendable) bool
+
+//AndBroadcastRule returns a new CanBeBroadcasted rule merging many rules witn AND
+func AndBroadcastRule(rules ...CanBeBroadcasted) CanBeBroadcasted {
```

Comment:
i wouldn't put this functionality here. If someone has a more complex rule, then it's their thing to do